dcos/subcommand.py

```diff
@@ -141,7 +141,7 @@ def default_subcommands():
     :returns: list of all the default dcos cli subcommands
     :rtype: [str]
     """
-    return ["auth", "config", "help", "marathon",
+    return ["auth", "config", "help", "job", "marathon",
```

Comment:
@tamarrow I'm not sure what you mean?  I have an entry for job in `_default_modules`